### DISCUSSION TITLE
Path to page is clickable but returns "This page does not exist yet"

### DISCUSSION THREAD

**Discussion by bronf** [requarks/wiki#5913] [Error / Bug Report] [answered]
At the top of a page, you can see the full path to the page like this:
`Home` / `folder1` / `folder2` / `page_file_name`

It is possible to click on `folder1` and `folder2` but this triggers an error _Page Not Found - This page does not exist yet - Would you like to create it now?_

The funny thing is that it is possible to create a page with the same name as a folder, namely `Home` / `folder1` / `folder2`.

But this creates very strange side effects: the tree view (for example when you want to move a page) shows the folder names as the page title of page having the same path.
For example if page `Home` / `folder1` / `folder2` exists and has title `The folder 2 page`, in the _move page_ window, you will see the folders as `Home` / `folder1` / `The folder 2 page`.

One thing that would be great would be that the folders are clickable as today but if no page exist for that path, it would show the content of the folder, ie page title and description.

**Comment by NGPixel** [comment, marked as answer]
This is as designed, not a side effect. A folder will always take the name of the page at its path (as folders don't technically exist).

The possibility to list children pages is coming in a future release.

**Comment by bronf** [comment]
OK, thanks. Sorry for misunderstanding this feature then. Waiting for the next release. For reference, it is explained in the documentation here : https://docs.requarks.io/guide/structure

**Comment by jaspercram** [comment]
I think there is still some confusion in the "main" page of a  directory. The issue is very well described in https://requarks.canny.io/wiki/p/show-tree-of-subdir-with-entrypages . I would propose a different solution from the "file with the same name as the directory" solution:

Choose a file name (like 'main', 'index' or 'home') als the default for directories. Show this page when people click on a directory in tree navigation and also show this page for clicks on the breadcrumb.
Some remarks:
- This also reduces confusion about the (side wide) homepage as it is the the index page of the root dir.
- If there is a page with a name identical to a dir, just treat it as any other page in the directory. (if this solution is implemented, it is quite unlikely that people with make such a page as it doesn't make much sense.)
- Maybe we want the name to be configurable in the settings.
- Obviously this is also how html does it. It doesn't harm reusing a proven concept.....
